src/order/filter/pointwise.lean

```diff
@@ -246,7 +246,10 @@ variables [has_div α] {f f₁ f₂ g g₁ g₂ h : filter α} {s t : set α} {a
 
 /-- The filter `f / g` is generated by `{s / t | s ∈ f, t ∈ g}` in locale `pointwise`. -/
 @[to_additive "The filter `f - g` is generated by `{s - t | s ∈ f, t ∈ g}` in locale `pointwise`."]
-protected def has_div : has_div (filter α) := ⟨map₂ (/)⟩
+protected def has_div : has_div (filter α) :=
+/- This is defeq to `map₂ (/) f g`, but the hypothesis unfolds to `t₁ / t₂ ⊆ s` rather than
+`set.image2 (/) t₁ t₂ ⊆ s`. -/
```

Comment:
Perhaps worth making clear that the latter is the more-unfolded version of the former
```suggestion
all the way to `set.image2 (/) t₁ t₂ ⊆ s`. -/
```
(and throughout)